src/basic/audit.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
BTW, I'd like to name like the following:
- basic/audit-util.h
- shared/libaudit-util.h